internal/meta/xmp.go

```diff
@@ -57,5 +57,14 @@ func (data *Data) XMP(fileName string) (err error) {
 		data.LensModel = doc.LensModel()
 	}
 
+	if takenAt := doc.DateCreated(); !takenAt.IsZero() {
+		data.TakenAt = takenAt
```

Comment:
How to set the local time here? With this implementation local time and UTC are the same?